ibm/service/power/resource_ibm_pi_volumes_attach.go

```diff
@@ -0,0 +1,99 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	st "github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceIBMPiVolumesAttach() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPiVolumesAttachCreate,
+		ReadContext:   resourceIBMPiVolumesAttachRead,
+		DeleteContext: resourceIBMPiVolumesAttachDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			Arg_CloudInstanceID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Cloud Instance ID of a PCloud Instance.",
```

Comment:
It's probably too much, but i've been sorting the fields withing the stanza as well for the data source refactor, and adding a comment at the beginning of the argument and attribute sections saying 
```
// Arguments 
```
and 
```
// Attributes
```
respectively to make a little easier to parse.